src/locales/en.json

```
@@ -1,13 +1,23 @@
 {
   "Add a Product" : "Add a Product",
   "Add to Shipment" : "Add to Shipment",
+  "Cancel": "Cancel",
+  "Click the backdrop to dismiss.": "Click the backdrop to dismiss.",
+  "History":"History",
+  "Inventory can be received for purchase orders in multiple shipments. Proceeding will recieve a new shipment for this purchase order but it will still be available for receiving later": "Inventory can be received for purchase orders in multiple shipments. { space } Proceeding will recieve a new shipment for this purchase order but it will still be available for receiving later",
   "item":"item",
   "items":"items",
   "Load more shipments":"Load more shipments",
   "Logout":"Logout",
+  "Proceed": "Proceed",
+  "Purchase Order":"Purchase Order",
+  "Purchase Order Details":"Purchase Order Details",
   "ReceiveAll": "ReceiveAll",
```

Is it intentional?
```suggestion
  "Receive All": "Receive All",
```

I can clearly see a space
https://www.figma.com/file/bVPRRw282CqGKMdbz7dciH/HC-Ionic-design-system?node-id=13525%3A356

@dt2patel

---

Yes Sir.